public/app/core/services/context_srv.ts

```diff
@@ -57,6 +58,14 @@ export class ContextSrv {
     store.set('grafana.sidemenu.pinned', val);
   }
 
+  setActiveState(val) {
+    this.active = val;
+  }
+
+  isActive() {
+    return this.active;
```

Comment:
is not easier if this returned document.visibilityState === 'visible' ? 

And the function should be isGrafanaVisible  ? because it does not seem to relate to activity as that state is triggered by user in activity, and user in-activity should not disable refresh (as grafana on tv's have little user activity but needs to refresh). So I think this function name is a bit miss leading